tests/test_fields.py

```diff
@@ -678,6 +678,17 @@ class F(Form):
             "</ul>"
         )
 
+    def test_render_kw_preserved(self):
+        F = make_form(
+            a=RadioField(
+                choices=[(True, "yes"), (False, "no")],
+                render_rw=dict(disabled=True)
```

Comment:
There is a typo here `render_rw` -> `render_kw`.

Reply:
Thank you! Fixed.